scripts/download.sh

```diff
@@ -34,6 +34,27 @@ else
   echo "Snapshot already present"
 fi
 
-if [ ! -z "$STATS_IP" ]; then
-  echo -n "ctlstore.reflector.init_snapshot_download_time:$(($END - $START))|h|#$TAGS" | nc -u -w1 $NODE_IP $STATS_PORT
-fi
\ No newline at end of file
+emit_metrics(){
+  if [ ! -z "$STATS_IP" ]; then
+    counter=0
+    emit=true
+    while ! echo exit | nc -u -w1 $NODE_IP $STATS_PORT;
+    if (($((counter % 15)) == 0)); then
+      echo "awaiting datadog UDP port to be ready..."
+    fi
+    counter=$((counter+1))
+    if ((counter > 360)); then
+      emit=false
```

Comment:
I would simplify this and simply emit the metric regardless if the timer has fully elapsed, getting rid of `emit`